docs/src/design/decoder/constraints.md

```diff
@@ -326,17 +326,23 @@ $$
 v_{dyn} = f_{dyn} \cdot ch_{dyn}  \text{ | degree} = 6
 $$
 
+When the `CALL` or `SYSCALL` operation is executed, the hash of the callee is added to the block hash table.
+
+$$
+v_{call\_or\_syscall} = (f_{call} + f_{syscall}) \cdot ch_1  \text{ | degree} = 5
+$$
+
 When `END` operation is executed, hash of the completed block is removed from the block hash table. However, we also need to differentiate between removing the first and the second child of a *join* block. We do this by looking at the next operation. Specifically, if the next operation is neither `END` nor `REPEAT` we know that another block is about to be executed, and thus, we have just finished executing the first child of a *join* block. Thus, if the next operation is neither `END` nor `REPEAT` we need to set the term for $\alpha_6$ coefficient to $1$ as shown below:
```

Comment:
We also need to update this paragraph to mention the `HALT` operation there.

Reply:
fixed